features/lang-attr.yml

```
@@ -0,0 +1,6 @@
+name: Lang
+description: The `lang` global attribute and property defines the language of an element.
```

Can we say a bit more about what this is useful for? Maybe:

```suggestion
description: The `lang` global attribute defines the language of an element, for the purpose of assistive technology to correctly read the content.
```